db/migrate/1_create_spina_tables.rb

```diff
@@ -1,6 +1,8 @@
 class CreateSpinaTables < ActiveRecord::Migration[4.2]
-  def change
-    create_table "spina_accounts", force: :cascade do |t|
+  def up
+    enable_extension "uuid-ossp"
```

Comment:
no idea why this was required, maybe `ActiveRecord::Migration[4.2]` doesn't know about `gen_random_uuid()`.